00-Raster-Structure.Rmd

```diff
@@ -334,18 +321,24 @@ plotRGB(newRGBImage, r = 1, g = 2, b = 3,
 ```
 
 
-If we are lucky, our geotiff file has a tag that tells us what the NoDATA value is.
-If we are less lucky, we can find that information in the raster's `metadata`. 
-If a NoDataValue was stored in the `geotiff` tag, when R opens up the raster,
-it will assign each instance of that value to `NA` (no data in r world). Values
+Several different NoData values are in common use (NA, NaN, -9999). If we are 
+lucky, our geoTIFF file has a tag that tells us what the NoData value is. If we 
+are less lucky, we can find that information in the raster's metadata.
+If a NoDataValue was stored in the GeoTIFF tag, when R opens up the raster,
+it will assign each instance of that value to `NA` (no data in R world). Values
 of `NA` will be ignored by R.
 
-#Add no data values to geotiff, then add section looking for -9999 values and
-#changing to NaN (or equivalent); there would be a challenge here, because the
-#-9999 should show up as the min value
+#Explain -9999 
 
-```{r no-data-values }
 
+# section looking for -9999 values 
+#Challenge:  -9999 should show up as the min value
+```{ r looking-for-9999-values}
+
+```
+
+#Convert to NaN (or equivalent)
+```{r convert-no-data-values }
```

Comment:
So GDAL info is a tool to view attributes of a file before opening it. it's not just for no data values. We can use OGR info for vector data. it's a good workflow to know. but boring stuff. where should it go? it could be a challenge potentially??